### PR TITLE
Signatures property in transation not checked for empty signature - Closes #5021

### LINKED ISSUE #5021: Signatures property in transation not checked for empty signature
### Expected behavior

Should invalidate `''` signature in `signatures`in `12_multisignature_transaction`

### Actual behavior

Not invalidated and causes error `Argument must be a valid hex string.`

### Which version(s) does this affect? (Environment, OS, etc...)
4.0-

### PR DESCRIPTION
**Pull request by pablitovicente**
### What was the problem?

This PR resolves #5021

### How was it solved?

By adding a check for empty string in the `signatures` array for multi signature registration/

### How was it tested?

Unit test added

### CODE REVIEW

**Comment by pablitovicente** [comment]
> We also need to change for base transaction, in that case, we should not call `validateSignature` for the empty string

This is already done in https://github.com/LiskHQ/lisk-sdk/blob/7809e36d3c1d0cc8ad31aace6b479253a091f660/elements/lisk-transactions/src/utils/verify.ts#L159

**Comment by pablitovicente** [comment]
> > We also need to change for base transaction, in that case, we should not call `validateSignature` for the empty string
> 
> This is already done in
> 
> https://github.com/LiskHQ/lisk-sdk/blob/7809e36d3c1d0cc8ad31aace6b479253a091f660/elements/lisk-transactions/src/utils/verify.ts#L159

had to fix `validateKeysSignatures` so the comment was correct

File: elements/lisk-transactions/src/utils/verify.ts [hunk 1]
```diff
@@ -98,8 +98,20 @@ export const validateKeysSignatures = (
 	transactionBytes: Buffer,
 ) => {
 	const errors = [];
+
 	// tslint:disable-next-line: prefer-for-of no-let
 	for (let i = 0; i < keys.length; i += 1) {
+		if (signatures[i].length === 0) {
+			errors.push(
+				new TransactionError(
+					'All mandatory keys have to sign the transaction.',
```
Comment (shuse2) [inline elements/lisk-transactions/src/utils/verify.ts]:
```suggestion
					'Invalid signatures format. signatures should not include empty string.',
```

Comment (shuse2) [reply elements/lisk-transactions/src/utils/verify.ts]: Also, since it's array, we can `continue` instead of `break`

Comment (pablitovicente) [reply elements/lisk-transactions/src/utils/verify.ts]: but as this are mandatory it doesn't make sense to continue as all need to be present.

**Pull request merged**